src/pages/index.tsx

```diff
@@ -140,33 +146,29 @@ const IndexPage = () => {
           <a href="https://develop.sentry.dev/self-hosted/">
             Self-Hosted Sentry
           </a>
-          <a href="https://help.sentry.io/">
-            Support
-          </a>
+          <a href="https://help.sentry.io/">Support</a>
         </div>
 
         <h3>Learn more...</h3>
         <div className="flex-row link-row">
           <div>
             <ul className="unstyled-list">
               <li>
-                <a href="/product/performance/">
-                  Performance Monitoring
-                </a>
+                <a href="/product/performance/">Performance Monitoring</a>
               </li>
               <li>
                 <a href="/product/sentry-basics/tracing/distributed-tracing/">
                   Distributed Tracing
                 </a>
               </li>
               <li>
-                <a href="/product/releases/health/">Release Health</a>
+                <a href="/product/session-replay/">Session Replay</a>
```

Comment:
We'll need the landing page before we can merge, the 404 checker will break the build

Reply:
Just read the bottom of the description 🚀